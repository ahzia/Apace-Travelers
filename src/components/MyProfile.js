const MyProfile = () => (
  <p>MyProfile</p>
);
export default MyProfile;
