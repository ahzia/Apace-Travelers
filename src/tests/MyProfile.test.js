import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyProfile from '../components/MyProfile';
import store from '../redux/configureStore';

describe('Component test snapshot', () => {
  it('Profile renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MyProfile />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
