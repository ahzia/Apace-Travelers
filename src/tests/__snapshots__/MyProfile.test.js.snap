// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component test snapshot Profile renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-md-3"
    >
      <div>
        <h3
          className="joined__mission__title"
        >
          No Missions joined!
        </h3>
        <div
          className="joined__mission__container"
        />
      </div>
    </div>
    <div
      className="col-md-3"
    >
      <div>
        <h3
          className="joined__mission__title"
        >
          No Rockets Reserved!
        </h3>
        <div
          className="joined__mission__container"
        />
      </div>
    </div>
    <div
      className="col-md-3"
    >
      <div>
        <h3
          className="joined__mission__title"
        >
          No Dragons Reserved!
        </h3>
        <div
          className="joined__mission__container"
        />
      </div>
    </div>
  </div>
</div>
`;
